tests/e2e/docs/vanilla_cluster_setup.md

```diff
@@ -60,6 +60,11 @@ list of datastore URLs where you want to deploy file share volumes. Retrieve thi
     # Make sure env var FULL_SYNC_WAIT_TIME should be at least double of the manifest variable FULL_SYNC_INTERVAL_MINUTES in csi-driver-deploy.yaml
     export FULL_SYNC_WAIT_TIME=350    # In seconds
     export USER=root
+    # Set VC,VC UI,ESX and K8S VM password accordingly
```

Comment:
instead writing this kind of password format  we can just mention <esx_password>, <vc_root_password>. etc..